CHANGES.rst

```diff
@@ -14,6 +14,7 @@ Enhancements and Fixes
 - Add a UAT constraint to the registry interface for constraining
   subjects [#649]
 
+- Fixed AttributeError when a capability has None standardID in SIA2Service. [#668]
```

Comment:
no big deal, but the reason this fails the tests is that we do include the reference number for the PR and not for the issue(s)